From: Christopher Obbard <obbardc@debian.org>
Date: Fri, 10 Apr 2026 13:33:14 +0100
Subject: Cargo.toml: drop pyo3 generate-import-lib feature

Remove the pyo3 generate-import-lib feature from Cargo.toml.

This feature is used for generating Python import libraries when
targeting Windows and is not needed for Debian builds on Linux.

Forwarded: not-needed
Signed-off-by: Christopher Obbard <obbardc@debian.org>
---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 9cf0dcc..4238825 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -13,7 +13,7 @@ path = "spice_crypt/pspice/_aes_brute.rs"
 crate-type = ["cdylib"]
 
 [dependencies]
-pyo3 = { version = "0.28", features = ["abi3-py310", "generate-import-lib"] }
+pyo3 = { version = "0.28", features = ["abi3-py310"] }
 rayon = "1"
 cpufeatures = "0.2"
 
